numba/np/arrayobj.py

```diff
@@ -1563,6 +1563,16 @@ def array_T(context, builder, typ, value):
     return impl_ret_borrowed(context, builder, typ, res)
 
 
+@overload(np.logspace)
+def numpy_logspace(start, stop, num=50, base=10.0):
+
```

Comment:
There's no type checking being done on the types of the arguments in the current implementation, perhaps take a look at the guide to using `@overload` for some idea of what is expected https://numba.readthedocs.io/en/stable/extending/overloading-guide.html#concrete-example (or just look at `np.rot90` immediately below this function)?

Reply:
Appreciate your advices, i have updated this part of code.